Battleship/Battleship/Game.cs

```diff
@@ -1,80 +0,0 @@
-﻿using System;
-
-namespace Battleship
-{
-    public class Game
-    {
-        public Player Player1 { get; set; }
-        public Player Player2 { get; set; }
-        private int roundCount { get; set; }
-
-        public Game()
-        {
-            Player1 = new Player("Player 1");
-            Player2 = new Player("Player 2");
-            Player1.SetShips();
-            Player2.SetShips();
-            Console.WriteLine("Player 1 sips:");
-            Player1.ShowBoards();
-            Console.WriteLine("Player 2 sips:");
-            Player2.ShowBoards();
-        }
-
-        public void PlayGame()
-        {
-            while (!Player1.DidLost && !Player2.DidLost)
-            {
-                PlayRound();
-                roundCount++;
-            }
-            Player1.ShowBoards();
-            Player2.ShowBoards();
-            Console.WriteLine($"Player 1 destroyed: {Player2.DestroyedShips} ships");
-            Console.WriteLine($"Player 2 destroyed: {Player1.DestroyedShips} ships");
-            Console.WriteLine($"Number of rounds: {roundCount}");
-
-            if (Player1.DidLost)
-            {
-                Console.WriteLine("Player 2 won the game!");
-            }
-
-            if (Player2.DidLost)
-            {
-                Console.WriteLine("Player 1 won the game!");
-            }
```

Comment:
Method DisplaySummary could be introduced

Player1.ShowBoards();
            Player2.ShowBoards();
            Console.WriteLine($"Player 1 destroyed: {Player2.DestroyedShips} ships");
            Console.WriteLine($"Player 2 destroyed: {Player1.DestroyedShips} ships");
            Console.WriteLine($"Number of rounds: {roundCount}");

            if (Player1.DidLost)
            {
                Console.WriteLine("Player 2 won the game!");
            }

            if (Player2.DidLost)
            {
                Console.WriteLine("Player 1 won the game!");
            }

Reply:
ok <3